openadapt/share.py

```diff
@@ -0,0 +1,110 @@
+"""Copy a recording from one computer to another.
+
+Usage:
+    python -m openadapt.share send --recording_id=1 
+    python -m openadapt.share receive --wormhole_code=<wormhole_code>
+"""
+
+from zipfile import ZipFile, ZIP_DEFLATED
+import os
+import subprocess
+
+from loguru import logger
+import fire
+
+from openadapt import config, crud, utils
+
+
+LOG_LEVEL = "INFO"
+utils.configure_logging(logger, LOG_LEVEL)
+
+
+def export_recording_to_folder(recording_id):
+    """Export a recording to a zip file.
+
+    Args:
+        recording_id (int): The ID of the recording to export.
+
+    Returns:
+        str: The path of the created zip file.
+    """
+    recording_db_path = crud.export_recording(recording_id)
```

Comment:
Please add a newline after docstring 🙏

Reply:
@abrichr I removed the doctstring because flake8-docstyle throws D202 error. There is some discussion online but In [PEP 257](https://peps.python.org/pep-0257/#multi-line-docstrings), it mentions to have an extra line after docstring for a class but not for a function. They also have an example below with a function with no space after docstring
Ex:
```python
def complex(real=0.0, imag=0.0):
    """Form a complex number.

    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero
    ...
```